main.py

```diff
@@ -12,9 +12,10 @@
 
 
 async def main():
-    with monitor_zoom(sys.argv[1]) as zoom:
-        async with create_robot(secrets.creds, secrets.address) as robot:
-            audience = Audience(robot)
+    log_level = sys.argv[2] if sys.argv[2] else "INFO"
```

Comment:
I like the intention here, but I suspect this is going to throw exceptions about "index out of bounds". It's probably more like this:
```python
try:
    log_level = sys.argv[2]
except Index Error:
    log_level = "INFO"
```

...if we had a third argument, I'd say it's probably worthwhile to switch to using `argparse`. With only two, I can go either way: this isn't very complicated yet, but it could become so. I guess leave as-is for now, and we'll move to `argparse` in the future if relevant. It would be something like:
```python
parser = argparse.ArgumentParser()
parser.add_argument("url")
parser.add_argument("log_level", default="INFO")
args = parser.parse_args()
with monitor_zoom(args.url, args.log_level) as zoom:
    ...
```

Reply:
...or another way to keep it as a one-liner would be
```python
log_level = sys.argv[2] if len(sys.argv) >= 2 else "INFO"
```

Reply:
I love using argparse. I agree, definitely move to argparse if there's more customization happening. I think for now, I'm going to do `if len(sys.arv) == 3`, since if there's more, we're moving to argparse anyways.